File: arrayc#/array3.cs
using System;

namespace ConsoleApp1
{
    internal class Program
    {
        static void Main(string[] args)
        {
            
            int[] arr = { 3, 6, 5, 3, 5, 5, 4, 5, 4, 3, 2, 3, 6, 4 };
            int count = arr.Length;
            int max = arr[0];
            int min = arr[1];
            int force = 0;
            for (int i = 0; i < arr.Length; i++)
            {
                force = max - min;
                if (arr[i] > max)
                {
                    max = arr[i];
                }
                if (arr[i] < min)
                {
                    min = arr[i];
                }
            }
            Console.WriteLine(force);
        }
    }
}
    }
}

File: arrayjs/array3.js
let count=14;
let arr=[3,6,5,3,5,5,4,5,4,3,2,3,6,4];
let max=arr[0];
let min=arr[1];
let force=0;
for(i=0;i<arr.length;i++){
    force=max-min;
    if(arr[i]>max){
        max=arr[i];
    }if(arr[i]<min){
        min=arr[i];
    }
    }
console.log(force);